tsconfig.base.json

```diff
@@ -13,7 +13,7 @@
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
     "allowSyntheticDefaultImports": true,
-    "importsNotUsedAsValues": "error",
+    "verbatimModuleSyntax": false,
```

Comment:
We can't use it now because we use types as values for services / controllers injection, right?

Reply:
We actually can! A few files needed an update 😄